@here/harp-mapview/lib/VisibleTileSet.ts

```diff
@@ -534,6 +539,9 @@ export class VisibleTileSet {
                     numTilesLoading++;
                 } else {
                     tile.numFramesVisible++;
+                    // If this tile's data source is "covering" then other tiles beneath it have
+                    // their rendering skipped, see [[Tile.willRender]].
+                    this.skipOverlappedTiles(dataSource, tile);
```

Comment:
It looks like this code is assuming that the background data-source is processed first, so it will not work in other scenarios, right?!
I think it would also be good enough to only support the skipping of background tiles, i.e. have more concrete code like "if tile is covering, remove it from the visible background tiles'